### PR TITLE
The second Gradle task (Step 3)

### PR DESCRIPTION
**Pull request by a1exanddrovich**

### CODE REVIEW

File: core/build.gradle [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java'
+
+group 'org.example'
+version '1.0.0'
+
+repositories {
+    flatDir {
+        dir '../libs'
```
Comment (NatalliaKhadunai) [inline core/build.gradle]: As we've already discussed we should load the library from the maven local repository, not from the `lib` folder which doesn't even exist here in the project.

File: core/src/main/java/com/epam/task/second/Utils.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.epam.task.second;
+
+import com.epam.task.first.StringUtils;
+
+public class Utils {
+
+    public boolean isAllPositiveNumbers(String... str) {
+        StringUtils utils = new StringUtils();
+        for (String number : str) {
```
Comment (NatalliaKhadunai) [inline core/src/main/java/com/epam/task/second/Utils.java]: Please implement this loop using Java 8 streams

File: api/build.gradle [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+repositories {
+    flatDir {
+        dir '../libs'
+    }
+}
+
+dependencies {
+    compile project(':core')
+    implementation name: 'utils-1.3.5'
```
Comment (NatalliaKhadunai) [inline api/build.gradle]: Why do we need to add `utils` library to the `api` project?

**Review by NatalliaKhadunai — COMMENTED** [review]
Left a couple of comments

File: core/src/main/java/com/epam/task/second/Utils.java (hunk 2 above)
Comment (a1exanddrovich) [reply core/src/main/java/com/epam/task/second/Utils.java]: Implemented

File: core/build.gradle (hunk 1 above)
Comment (a1exanddrovich) [reply core/build.gradle]: Fixed

**Pull request merged**